pkgs/development/tools/conftest/default.nix

```diff
@@ -0,0 +1,34 @@
+{ lib, buildGoModule, fetchFromGitHub }:
+
+let version = "0.7.0"; in
+
+buildGoModule {
+  pname = "helmfile";
+  inherit version;
+
+  src = fetchFromGitHub {
+    owner = "instrumenta";
+    repo = "conftest";
+    rev = "v${version}";
+    sha256 = "0qq2kp9h91rirlhml5vyzmi7rd4v3pkqjk2bn7mvdn578jnwww24";
+  };
+
+  buildFlagsArray = ''
+    -ldflags=
+        -X main.version=${version}
+        -X main.commit=751e61129f5703d9240d4c815bd8b79b5b4c6990
+        -X main.date=2019-06-16
```

Comment:
Please remove both date and commit. Is the version working as well?

Reply:
What's the motivation behind this suggestion? All three show up in `conftest --version`:
```
Version: 0.7.0
Commit: 751e61129f5703d9240d4c815bd8b79b5b4c6990
Date: 2019-06-16
```

In my opinion, it's better to include the information than to remove it. The defaults leave a lot to be desired: https://github.com/instrumenta/conftest/blob/v0.7.0/conftest.go#L41-L45

Reply:
Mainly because it's going to make your life updating it harder. IMO version (which is a tag anyway) is enough.